kafka/consumer.go

```diff
@@ -78,7 +69,7 @@ type ConsumerConfig struct {
 	//
 	// The processing time of each processing cycle can be calculated as:
 	// record.process.time * MaxPollRecords.
-	Processor model.BatchProcessor
+	Processor apmqueue.Processor
```

Comment:
The comment needs an update.

Reply:
Why does it need to be updated? The function hasn't changed.

Reply:
```
	// Processor that will be used to process each record individually.
	// It is recommended to keep the synchronous processing fast and below the
	// rebalance.timeout.ms setting in Kafka.
```
Maybe(?)
Also, not part of the PR, I don't quite get the `rebalance.timeout.ms` point, maybe there is a typo there.

Reply:
The processor will process each of the fetched record(s) individually. The comment is meant to inform users that passing a Processor which takes longer to process a single record than the rebalancing timeout in Kafka will potentially cause the record to be consumed twice (if a rebalance is waiting to happen while a record is being processed).

Reply:
Ah, thanks for the explanation. Now I can read the same line correctly, for some reason I kept reading `below` to mean beneath the line😅